### PR TITLE
Added Matrix_Sort.py

### PR DESCRIPTION
**Pull request by rohangawhade**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

Yes
### Description

Added Matrix_Sort.py file which sorts the content of the matrix while keeping the structure of matrix unchanged.
### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

#825

### CODE REVIEW

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@rohangawhade Update the code according to PEP 8 rules, check your code here http://pep8online.com/. Add time and space complexity.

**Comment by rohangawhade** [comment]
> @rohangawhade Update the code according to PEP 8 rules, check your code here http://pep8online.com/. Add time and space complexity.

Updated the code according to the PEP8 rules from the link shared. Also added time and space complexity at the end of the code. @iamrajiv

File: Python/sort/Matrix_Sort.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+# 8 3 5
+# 9 1 2
+# 4 7 6
+# After sorting:
+# 1 2 3
+# 4 5 6
+# 7 8 9
+
+
+# time complexity = O(n^2)
```
Comment (iamrajiv) [inline Python/sort/Matrix_Sort.py]: @rohangawhade I think the time complexity will be `O(n^2log2(n))` since you have used `sort()` function.

**Review by iamrajiv — CHANGES_REQUESTED** [review]

File: Python/sort/Matrix_Sort.py (hunk 1 above)
Comment (rohangawhade) [reply Python/sort/Matrix_Sort.py]: Okay. So I'll update the time complexity. @iamrajiv

File: Python/sort/Matrix_Sort.py [hunk 2]
```diff
@@ -0,0 +1,57 @@
+def sort_matrix(mat, size):
```
Comment (atarax665) [inline Python/sort/Matrix_Sort.py:1]: Add docstring to explain how the function works.

**Review by atarax665 — CHANGES_REQUESTED** [review]

File: Python/sort/Matrix_Sort.py (hunk 2 above)
Comment (rohangawhade) [reply Python/sort/Matrix_Sort.py:1]: Added the docstring and checked for PEP8 requirements too. @atarax665

**Review by iamrajiv — APPROVED** [review]

**Review by atarax665 — APPROVED** [review]
LGTM :)

**Review by SKAUL05 — APPROVED** [review]

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@rohangawhade Resolve the merge conflicts.

**Comment by rohangawhade** [comment]
> @rohangawhade Resolve the merge conflicts.

Done. @iamrajiv

**Review by iamrajiv — APPROVED** [review]

**Comment by HarshCasper** [comment]
@all-contributors bot please add @rohangawhade for code

**Pull request merged**